File: INVERTED TRIANGLE PATTERN/code8.c
#include <stdio.h>


void main(){

	int row ;
	printf("Enter no. of rows : ");
	scanf("%d",&row);

	for(int i = 0 ; i < row ; i++){

		for(int j = 0 ; j < row - i ; j++){


			printf("%d\t",((row)-j)*(row));

				
		}

		printf("\n\n");
	}

}

File: JAVA DAILYCODES ASSGNPRATICE QUIZE DSA/Basic is else assignments/11/code8.java

import java.util.Scanner;

class SpaceDemo{

	public static void main(String[] args){

		Scanner s = new Scanner(System.in);
		System.out.print("Enter rows : ");
		int row = s.nextInt();

		for(int i = 0 ; i < row ; i++){

			for(int k = 0 ; k < i; k++){

				System.out.print("		");
			}

			for(int j = 0; j < row - i ; j++){

				System.out.print((j+1+i)+"		");
			}

				System.out.print("\n\n");

		}

	}

}
